test/augmentation/test_augmentation.py

```diff
@@ -24,6 +25,794 @@
+        return request.param
+
+    def test_random_p_1(self):
+        torch.manual_seed(42)
+
+        input_tensor = torch.tensor([[[0.1, 0.2, 0.3, ],
+                                      [0.4, 0.5, 0.6, ],
+                                      [0.7, 0.8, 0.9, ]]], device=self.device, dtype=self.dtype)
+        expected_output = torch.tensor([[[[0.7, 0.8, 0.9, ],
+                                          [0.4, 0.5, 0.6, ],
+                                          [0.1, 0.2, 0.3, ]]]], device=self.device, dtype=self.dtype)
+
+        parameters = {}
+        self._test_random_p_1_implementation(
+            input_tensor=input_tensor,
+            expected_output=expected_output,
+            params=parameters)
+
+    def test_random_p_1_return_transform(self):
+        torch.manual_seed(42)
+
+        input_tensor = torch.tensor([[[0.1, 0.2, 0.3, ],
+                                      [0.4, 0.5, 0.6, ],
+                                      [0.7, 0.8, 0.9, ]]], device=self.device, dtype=self.dtype)
+        expected_output = torch.tensor([[[[0.7, 0.8, 0.9, ],
+                                          [0.4, 0.5, 0.6, ],
+                                          [0.1, 0.2, 0.3, ]]]], device=self.device, dtype=self.dtype)
+        expected_transformation = torch.tensor([[[1.0, 0.0, 0.0],
+                                                 [0.0, -1.0, 2.0],
+                                                 [0.0, 0.0, 1.0]]], device=self.device, dtype=self.dtype)
+        parameters = {}
+        self._test_random_p_1_return_transform_implementation(
+            input_tensor=input_tensor,
+            expected_output=expected_output,
+            expected_transformation=expected_transformation,
+            params=parameters)
+
+    def test_batch(self):
+        torch.manual_seed(12)
+
+        input_tensor = torch.tensor([[[[0.1, 0.2, 0.3, ],
+                                       [0.4, 0.5, 0.6, ],
+                                       [0.7, 0.8, 0.9, ]]]], device=self.device, dtype=self.dtype).repeat((2, 1, 1, 1))
+        expected_output = torch.tensor([[[[0.7, 0.8, 0.9, ],
+                                          [0.4, 0.5, 0.6, ],
+                                          [0.1, 0.2, 0.3, ]]]], device=self.device, dtype=self.dtype).repeat((2, 1, 1, 1))  # noqa: E501
+        expected_transformation = torch.tensor([[[1.0, 0.0, 0.0],
+                                                 [0.0, -1.0, 2.0],
+                                                 [0.0, 0.0, 1.0]]], device=self.device, dtype=self.dtype).repeat((2, 1, 1))  # noqa: E501
+        parameters = {}
+        self._test_random_p_1_return_transform_implementation(
+            input_tensor=input_tensor,
+            expected_output=expected_output,
+            expected_transformation=expected_transformation,
+            params=parameters)
+
+    @pytest.mark.skip(reason="No special parameters to validate.")
+    def test_exception(self):
+        pass
```

Comment:
I would just let to pass. Skipping I guess is for given some constrains or conditions.